routes.js

```diff
@@ -14,4 +14,59 @@ router.delete('/users',(req,res,next)=>{
         res.json(user);
     })
 })
-module.exports = router;
+
+      
+ //GET 
+
+ router.get('/leaderboard',(req,res)=>{
+        res.send("foobar");
+    });
+
+  router.get('/companylist',(req,res)=>{
+        res.send("foobar");
+    });
+  
+
+  router.get('/costumerdetail',(req,res)=>{
+        res.send("foobar");
+    });
+
+
+  router.get('/companydetail',(req,res)=>{
+        res.send("foobar");
+  }); 
+
+  router.get('/newslist',(req,res)=>{
+        res.send("foobar");
+  });
+
+  router.post('/buy',(req,res)=>{
+        res.send("foobar");
+    });
+
+
+  router.post('/sell',(req,res)=>{
+        res.send("foobar");
+    });
+
+
+  router.post('/short',(req,res)=>{
+        res.send("foobar");
+    });
+  
+
+  router.post('/cover',(req,res)=>{
+        res.send("foobar");
+    });
+
+
+  router.post('/takeloan',(req,rest)=>{
+        res.send("foobar");
+    });
+ 
+
+  router.post('/repayloan',(req,res)=>{   
+        res.send("foobar");
+    });
+
```

Comment:
Just a last change, please make the routes in snake_case. 
customer_detail, company_list etc. All the words which are actually two words, snake_case them